src/app.ts

```diff
@@ -19,6 +19,9 @@ const port = configuration.port;
 
 app.use("/", homeController.router);
 app.use("/seasons", seasonsController.router);
+
+// Allow Let's Encrypt to access challenge static content
+app.use("/.well-known/acme-challenge", express.static(__dirname + "/.well-known/acme-challenge"));

```

Comment:
how does this file get populated?

Reply:
There's an azure extension for Let's Encrypt.  It generates a challenge file in this directory whenever the SSL cert expires (every 3 months)